### PR TITLE
Add a test for conflicts in auto-user-instrum

### PR DESCRIPTION
**Pull request by manuel-alvarez-alvarez**
## Motivation

Check the behavior of the tracers when conflicting remote config payloads are received from the agent in regards to auto user instrumentation.

## Changes

Add new config files to the test triggering a conflict in the configuration of auto user instrumentation.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: utils/_remote_config.py [hunk 1]
```diff
@@ -27,31 +27,34 @@ def send_command(raw_payload) -> dict[str, Any]:
...
-    assert len(client_configs) == 1, "Only one client config is supported"
-    _, _, product, config_id, _ = client_configs[0].split("/")
-
-    config_state = {
-        "id": config_id,
-        "product": product,
-        "apply_state": ApplyState.UNKNOWN,
-        "apply_error": "<No known response from the library>",
-    }
+    current_states = {}
```
[the remark this answers is not in the document]
Comment (manuel-alvarez-alvarez) [reply utils/_remote_config.py:39]: Yep, done!

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
`./format.sh` :)

**Comment by manuel-alvarez-alvarez** [comment]
> `./format.sh` :)

Oops! sorry for that, my bad!, formatted :smile:

**Review by cbeauchesne — COMMENTED** [review]
your PR isn't an PR to main, do you still need a review ?

**Comment by manuel-alvarez-alvarez** [comment]
> your PR isn't an PR to main, do you still need a review ?

Nope, we are still discussing internally  if we want to have this kind of unspecified behavior tested (That's why I splited the PR in two). I've converted it to draft so it does not introduce noise for you.

**Comment by manuel-alvarez-alvarez** [comment]
No longer needed

**Pull request closed without merging**